### DISCUSSION TITLE
Is the principle about behavior towards each other appropriate?

### DISCUSSION THREAD

**Discussion by fienen** [DrunkenUX/web-ethics-commitment#13] [General]
> 6. I understand all designers or developers have varying levels of experience and exposure to certain techniques, technologies, and tools, and will not demean or belittle them for seeking out answers

The question has been raised if this makes sense as a principle, when all the other first draft items direct the behavior of the individual towards the user, rather than the individual towards their peers. While there's no question that we should treat each other with respect and kindness, and understand different experience levels, does this make sense to include in the bubble of outlining how we execute our trade ethically?

Alternatively, is there a rewording of the issue that might also make it align better in spirit with the other suggestions?

**Comment by AmirAsghary** [comment]
I don't believe the web-ethics commitment should be limited to dev-user relationships. So I think it should stay. However, since every other commitment starts with "I will", this one would need to start with it as well for better consistency. 
Maybe something like: "I will treat my peers and customers with respect and will not belittle them for seeking out answers" ?

**Reply by fienen** [reply]
True. What about something short and sweet, like:

> I will be respectful of my peers and users, and offer guidance and assistance when possible.

The rationale behind this can be explored in the subsequent page that explains each principle in more depth. The whole "everybody's new at something" mantra doesn't necessarily need baked into the up front text.

**Reply by AmirAsghary** [reply]
👌 Great.

**Reply by armahillo** [reply]
This seems better, yeah

I still think it might be good to have separate statements of commitment for each of Dev/User and Dev/Dev -- There are probably other affirmations that would make sense for Dev/Dev relations (looking at the current stuff going on re: the Ruby discussion list, this is clearly still a problem)

**Comment by fienen** [comment]
Converted this issue into a discussion.

**Reply by armahillo** [reply]
weeeiiirddd comments have upvotes now :-O
